src/lib/constants.js

```diff
@@ -1,15 +1,12 @@
+"use strict";
```

Comment:
This is done by Babel in transpile  time, it is good to have but not required. I would not use it unless you have a reason.

Reply:
Ohh, it is JS, we should remove this file.